app/ChordNode.js

```diff
@@ -428,7 +432,12 @@ class ChordNode {
       this.predecessor = this.encapsulateSelf();
     }
 
-    await this.migrateKeys();
+    try {
+      console.log("join: calling migrateKys");
+      await this.migrateKeysAfterJoin();
+    } catch (error) {
+      console.log("Migrate keys failed with error:", error);
```

Comment:
This should be console.error so it goes to the STDERR console.

Reply:
You already mentioned this, I forgot...

Reply:
Wait, does `console.err` work? I thought it was `console.error`? https://millermedeiros.github.io/mdoc/examples/node_api/doc/stdio.html#console.error

Reply:
You are right, typo. 
I guess when I tested it I never trigger that error and I didn't realize.
Thanks!